src/Modules/Finance/Header/Header.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';

```

Comment:
Could you please convert this component to a functional one? (using hooks)

Reply:
Yes, of course.

Reply:
let's do it later